07_errors/n0npax/errors.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+)
+
+// Error codes
+const (
+	ErrInvalidInputCode  = 400
+	ErrNotFoundCode      = 404
+	ErrInternalErrorCode = 500
+)
+
+// Error wrapps errors with code, message and error itself
+type Error struct {
+	Message string
+	Code    int
+}
+
+// Error returns error as a string
+func (e *Error) Error() string {
+	return fmt.Sprintf("Error code: %v, message : %v", e.Code, e.Message)
```

Comment:
Errors should not be formatted with a leading capital letter: https://github.com/golang/go/wiki/Errors

I'd also suggest that generally the error code doesn't mean much to the user, so I wouldn't print it in the message - although in this case because you're using http status code, everyone knows them, right :). 

I would just format the error string as `"%v (%v)", e.Message, e.Code` (if you want to keep the code. The most important part is the message, but you've put that at the end. Lead with that instead.

Reply:
done